test/spec/features/context-pad/ContextPadProviderSpec.js

```diff
@@ -291,6 +291,14 @@ describe('features - context-pad', function() {
       ]);
     }));
 
+    it('should provide text annotation entries', inject(function() {
+      expectContextPadEntries('TextAnnotation_1', [
+        'connect',
+        'delete',
+        '!replace'
```

Comment:
We should also explicitly check, that you cannot annotate given you have an annotation

```suggestion
        '!replace',
        '!append.text-annotation'
```